runtime/src/weights/pallet_teerex.rs

```diff
@@ -52,15 +51,15 @@ impl<T: frame_system::Config> pallet_teerex::WeightInfo for WeightInfo<T> {
 	// Storage: Teerex EnclaveIndex (r:1 w:0)
 	// Storage: Teerex WorkerForShard (r:0 w:1)
 	// Storage: Teerex LatestIpfsHash (r:0 w:1)
-	fn confirm_call() -> Weight {
+	fn confirm_processed_parentchainblock() -> Weight {
```

Comment:
That one is already outdated again. But I'll have to do an upgrade to pallet_teerex anyway to finish https://github.com/integritee-network/worker/issues/457, so leave it for now. We just need to make sure we don't downgrade once we're ready to merge this one.